shoten/cli.py

```diff
@@ -37,9 +37,9 @@ def parse_args(args: Any) -> Any:
 def process_args(args: Any) -> None:
     'Process input according to CLI arguments.'
     if args.read_file:
-        myvocab = load_wordlist(args.read_file, langcodes=tuple(args.language))
+        myvocab = load_wordlist(args.read_file, langcodes=tuple(args.language), maxdiff=10000)
     elif args.read_dir:
-        myvocab = gen_wordlist(args.read_dir, langcodes=tuple(args.language))
+        myvocab = gen_wordlist(args.read_dir, langcodes=tuple(args.language), maxdiff=10000)
```

Comment:
Really weird case - I initially thought this was another one of those comments I hate where we have questioned an intentional change, but looking at the readme it looks like this might be wrong! 

In the Readme it references that the default value for maxdiff should be 1000, but here it sets it to 10000. It would be good to reference that difference in the comment to make it clear why we're questioning things